sycl/test/devicelib/std_complex_math_test.cpp

```diff
@@ -1,9 +1,7 @@
-// UNSUPPORTED: windows
-// RUN: %clangxx -fsycl -c %s -o %t.o
-// RUN: %clangxx -fsycl %t.o %sycl_libs_dir/libsycl-complex.o %sycl_libs_dir/libsycl-cmath.o -o %t.out
+// REQUIRES: cpu, linux
```

Comment:
Delete line.

Reply:
Fixed.

Reply:
Hi, @vzakhari 
I have reverted the "ACC_RUN" in devicelib lit tests. And to math/complex tests, we have 2 issues:
1. Some MSVC float math functions's implementation will use double, so although those functions are float, they must work with fp64 extension support.(ldexpf, frexpf have this problem). And some float complex math functions will depend on ldexpf, so those complex math functions have the same issue. I have splitted all math tests into 2 parts, and those functions with this "fake float" issue will only run on Linux(Linux implementation is nice, no "fake float").
2. Some math constants such as M_PI can only be used with "-D_USE_MATH_DEFINES" in MSVC, I have commented in the test.
Thank you very much.

Reply:
Thank you.